src/java.base/bsd/native/libjava/ProcessHandleImpl_bsd.c

```diff
@@ -23,6 +23,10 @@
  * questions.
  */
 
+#ifdef __NetBSD__
+#define _KMEMUSER
```

Comment:
Why this? Do you run the process as root and read kernel memory?

Reply:
It looks like NetBSD transitioned to KERN_PROC2 with struct kinfo_proc2 which is similar to the 6 entry mib that OpenBSD uses as well.

Reply:
This is a fair question.  Without this there are compilation errors, but let me see if I can find a better solution.

Reply:
I've checked again and the final version of these changes didn't need _KMEMUSER.  I just needed to include <sys/resource.h>.  I've done so and removed the _KMEMUSER define.